src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+const calendar = document.querySelector('.calendar');
+
+const daysInMonth = 31;
+
+function generateCalendar(daysInMonth) {
+
+  // Add divs for each day in the month
+  for (let i = 1; i <= daysInMonth; i++) {
```

Comment:
The variable `i` is not descriptive. Consider using a more descriptive name like `dayNumber`.